go/kbfs/kbfsgit/runner.go

```diff
@@ -1889,6 +1921,55 @@ func (r *runner) handleOption(ctx context.Context, args []string) (err error) {
 	return err
 }
 
+func (r *runner) copyFileLFS(
+	ctx context.Context, from billy.Filesystem, to billy.Filesystem,
+	fromName string, toName string) (err error) {
+	f, err := from.Open(fromName)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	toF, err := to.Create(toName)
+	if err != nil {
+		return err
+	}
+	defer toF.Close()
+
+	// TODO: wrap the writer
+	var w io.Writer = toF
+	_, err = io.Copy(w, f)
```

Comment:
Why do we need to wrap the writer?

Reply:
For progress messages (just merged as #20918).